deploy-disconnected-registry/olm-sync.sh

```diff
@@ -74,16 +74,8 @@ function mirror() {
     registry_login ${SOURCE_REGISTRY}
     echo ">>>> Podman Login into Destination Registry: ${DESTINATION_REGISTRY}"
     registry_login ${DESTINATION_REGISTRY}
+    cp -rf ${PULL_SECRET} ~/.docker/config.json
```

Comment:
since $PULL_SECRET is a single file, no need to append the -r param, i.e. 'cp -f' is sufficient.